tests/integ/modin/conftest.py

```diff
@@ -45,6 +45,7 @@ def pytest_addoption(parser):
     parser.addoption(
         "--generate_pandas_api_coverage", action="store_true", default=False
     )
+    parser.addoption("--skip_sql_count_check", action="store_true", default=False)
```

Comment:
can we have a test for this?

Reply:
hmm what's the best way to test it? I'll use this argument very soon in #1626